### PR TITLE
Make the returned alpha value a float number

### LINKED ISSUE #3: Alpha should be a float
Curious why the returned alpha value is not divided by 255? Just by looking at the code the returned number is still relative to hex and not an alpha percentage / fraction as inputed by rgb-hex.

### PR DESCRIPTION
**Pull request by molnarmark**
This PR fixes #3.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you update the tests and readme too? :)

**Comment by molnarmark** [comment]
Yep! Just done that.

File: index.js [hunk 1]
```diff
@@ -16,12 +16,12 @@ module.exports = function (hex, options = {}) {
 	let alpha = 255;
 
 	if (hex.length === 8) {
-		alpha = parseInt(hex.slice(6, 8), 16);
+		alpha = parseFloat((parseInt(hex.slice(6, 8), 16) / 255).toFixed(4));
```
Comment (sindresorhus) [inline index.js]: Why are you limiting it to 4 fractional digits? I don't really see the benefit except for presentation. This needlessly limits the accuracy.

Comment (molnarmark) [reply index.js]: I honestly have no idea why I've done that, mistake on my end, sorry about that. 😄

File: index.js [hunk 2]
```diff
@@ -16,12 +16,12 @@ module.exports = function (hex, options = {}) {
 	let alpha = 255;
 
 	if (hex.length === 8) {
-		alpha = parseInt(hex.slice(6, 8), 16);
+		alpha = parseFloat((parseInt(hex.slice(6, 8), 16) / 255));
```
Comment (sindresorhus) [inline index.js]: The `parseFloat` call is moot now

File: readme.md [hunk 3]
```diff
@@ -25,13 +25,13 @@ hexRgb('#fff');
 //=> {red: 255, green: 255, blue: 255, alpha: 255}
 
 hexRgb('#4183c488');
-//=> {red: 65, green: 131, blue: 196, alpha: 136}
+//=> {red: 65, green: 131, blue: 196, alpha: 0.5333333333333333}
```
Comment (sindresorhus) [inline readme.md]: Can you use a HEX alpha that doesn't end up with a lot of fractions? Better for presentational purposes.

File: readme.md [hunk 4]
```diff
@@ -24,14 +24,14 @@ hexRgb('#4183c4');
 hexRgb('#fff');
 //=> {red: 255, green: 255, blue: 255, alpha: 255}
 
-hexRgb('#4183c488');
-//=> {red: 65, green: 131, blue: 196, alpha: 136}
+hexRgb('#22222299');
+//=> {red: 65, green: 131, blue: 196, alpha: 0.6}
```
Comment (sindresorhus) [inline readme.md]: The red/green/blue are incorrect

File: readme.md [hunk 5]
```diff
@@ -24,14 +24,14 @@ hexRgb('#4183c4');
...
 
-hexRgb('#0008');
-//=> {red: 0, green: 0, blue: 0, alpha: 136}
+hexRgb('#cd2222cc');
+//=> {red: 0, green: 0, blue: 0, alpha: 0.8}
```
Comment (sindresorhus) [inline readme.md]: Same here

File: readme.md [hunk 6]
```diff
@@ -24,14 +24,14 @@ hexRgb('#4183c4');
...
 //=> {red: 255, green: 255, blue: 255, alpha: 255}
 
-hexRgb('#4183c488');
-//=> {red: 65, green: 131, blue: 196, alpha: 136}
+hexRgb('#22222299');
+//=> {red: 65, green: 131, blue: 196, alpha: 0.6}
 
-hexRgb('#0008');
-//=> {red: 0, green: 0, blue: 0, alpha: 136}
+hexRgb('#cd2222cc');
```
Comment (sindresorhus) [inline readme.md]: This one was intentionally using the short format, so show that the short format also works with alpha.

Comment (molnarmark) [reply readme.md]: This one was removed due to your previous problem with the fraction digits.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**